modules/microshift-lvm-thin-volumes.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * microshift_storage/creating-volume-snapshots-microshift.adoc
+
+:_content-type: CONCEPT
+[id="microshift-lvm-thin-volumes_{context}"]
+= About LVM thin volumes
+
+To use advanced storage features such as creating volume snapshots or volume cloning, you must provision a logical volume manager (LVM) thin-pool on the {op-system-ostree} host and configure both the logical volume manager storage (LVMS) provider and the cluster. LVM thin-pools must be attached to a volume group.
+
+[IMPORTANT]
+====
+To create CSI snapshots, you must configure thin volumes on the {op-system-ostree} host. The CSI does not support volume shrinking because of the potential for data loss.
+====
+
+For LVMS to manage thin logical volumes (LVs), a thin-pool `device-class` must be specified in the `lvmd.yaml` file. Multiple thin-pool device classes are permitted.
+
+.Example `lvmd.yaml` that specifies a single device class for a thin-pool
+
+[source, yaml]
+----
+socket-name: <1>
+device-classes: <2>
+  - name: thin <3>
+    default: true
+    spare-gb: 0 <4>
+    thin-pool:
+      name: thin
+      overprovision-ratio: 10 <5>
+    type: thin
+    volume-group: ssd <6>
+    stripe: <7>
+    stripe-size: <8>
+    lvcreate-options: <9>
```

Comment:
Stripe, stripe-size, and lvcreate-options are not related specifically to thin volumes, so we could leave those out of the example.

Reply:
@copejon You said these were needed (in the google doc). Can you weigh in?

Reply:
Right, the options aren't specific to thin vols.  IIRC the inclusion of these fields was related to a broader config example. I would leave them out of this example.